sdk/storage/Azure.Storage.Common/src/StorageChecksumAlgorithm.cs

```diff
@@ -15,18 +15,18 @@ public enum ValidationAlgorithm
         Auto = 0,
 
         /// <summary>
-        /// No selected algorithm.
+        /// No selected algorithm. Do not calculate or request checksums.
         /// </summary>
         None = 1,
 
         /// <summary>
-        /// Azure Storage custom 64 bit CRC.
+        /// Standard MD5 hash algorithm.
         /// </summary>
-        StorageCrc64 = 2,
+        MD5 = 2,
 
         /// <summary>
-        /// Standard MD5 hash algorithm.
+        /// Azure Storage custom 64 bit CRC.
```

Comment:
Can we add a link for people to learn more?

Reply:
There is not currently public documentation. Since we use a custom polynomial, there is an expectation that anyone using this field is letting the client calculate and set it instead of reimplementing 64bit crc themselves.